packages/webpack-config-single-spa-react/package.json

```diff
@@ -27,9 +27,6 @@
     "url": "https://github.com/CanopyTax/create-single-spa/issues"
   },
   "dependencies": {
-    "clean-webpack-plugin": "^3.0.0",
```

Comment:
These are dependencies of webpack-config-single-spa, so no need to add them again here